src/navigation/NavBar.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import AnchorLink from "react-anchor-link-smooth-scroll";
-import logo from "../assets/images/logo.png";
+import logo from "../assets/images/flags.png";
```

Comment:
You'll need to fix the size of the flags on the NavBar, they're way too big: 

<img width="1435" alt="Screen Shot 2019-07-09 at 11 38 48 PM" src="https://user-images.githubusercontent.com/38846104/60947163-c2ea9300-a2a4-11e9-88ff-fbf15aaaf76d.png">

Reference the old logo...replacing the old logo should not affect the height of the NavBar.